articles/machine-learning/how-to-log-view-metrics.md

```diff
@@ -172,24 +172,41 @@ import mlflow
 
 mlflow.config.enable_async_logging()
 ```
+It is recommended to use above global flag when you want to influence the logging behavior at all places and/or you are using some 
```

Comment:
instead of "influence the logging behavior at all places" - clarify to "enable asynchronous logging for training session"

Reply:
Fixed.